src/DetailsView/components/overview-content/overview-content-container.tsx

```diff
@@ -78,9 +78,9 @@ export const OverviewContainer = NamedFC<OverviewContainerProps>('OverviewContai
     );
 
     return (
-        <div className="overview">
+        <div className={styles.overview}>
```

Comment:
Test failure seems to be a result of test not being updated to reflect this change - overview.test.ts@37 is still looking for `overviewSelectors.overview` which is `'.overview'`, but now the style has a generated id like `'overview--1tASe'`

Reply:
Yep! Just needed to change to automation id.

Reply:
Just verified on my box that's it, got the test to pass using automation id.